packages/@adobe/spectrum-css-temp/components/table/skin.css

```diff
@@ -41,7 +41,7 @@ governing permissions and limitations under the License.
       }
     }
 
-    &:active {
+    &.is-active {
```

Comment:
doesn't :active get transformed to .is-active?

Reply:
no, only :hover